llvm/include/llvm/Analysis/BranchProbabilityInfo.h

```diff
@@ -343,6 +343,9 @@ class BranchProbabilityInfo {
   /// Keeps mapping of a loop to estimated weight to enter the loop.
   SmallDenseMap<LoopData, uint32_t> EstimatedLoopWeight;
 
+  /// Keeps mapping of a Loop to its "exit" blocks.
+  SmallDenseMap<LoopData, SmallVector<BasicBlock *, 4>> LoopExitBlocks;
```

Comment:
Can this be a local variable in computeEestimateBlockWeight? Just to make it clear that there are no invalidation concerns...

Reply:
Thanks! done.